FriendsAndPlaces/Models/Logout/LogoutRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace FriendsAndPlaces.Models.Logout
+{
+
+	public class LogoutRequest
+	{
+		[JsonProperty(PropertyName = "loginName")]
+		public string LoginName { get; set; }
+
+		[JsonProperty(PropertyName = "sessionId")]
```

Comment:
```suggestion
		[JsonProperty(PropertyName = "sitzung")]
```

Wenn man in eine Request-Beispiele schaut, wird beim Logout die Session als "sitzung" mitgeschickt.